src/Components/test/E2ETest/Tests/BootResourceCachingTest.cs

```diff
@@ -18,6 +18,9 @@
 
 namespace Microsoft.AspNetCore.Components.E2ETest.Tests
 {
+    // Disabling parallelism for these tests because of flakiness
+    [CollectionDefinition(nameof(BootResourceCachingTest), DisableParallelization = true)]
+    [Collection(nameof(BootResourceCachingTest))]
```

Comment:
Does this mean that this collection runs sequentially from any other collection and tests that are not in a collection?

Reply:
Yes, that is my understanding based on the Xunit docs.